docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-merchant-portal-core-feature-integration.md

```diff
@@ -54,7 +54,34 @@ Make sure that the following modules have been installed:
 
 {% endinfo_block %}
 
-### 2) Set up the database schema
+### 2) Set up configuration
+
+Add the following configuration:
+
+| CONFIGURATION                        | SPECIFICATION                                                                | NAMESPACE        |
+|--------------------------------------|------------------------------------------------------------------------------|------------------|
+| GuiTableConfig::getDefaultTimezone() | Defines default timezone for formatting `DateTime` data to ISO 8601 format.. | Pyz\Zed\GuiTable |
+
```

Comment:
please add the file name, example: `**src/Pyz/Zed/Merchant/Persistence/Propel/Schema/spy_merchant.schema.xml**`